build/docker/prometheus-linux.yml

```diff
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 5s
+  evaluation_interval: 60s
+
+scrape_configs:
+  - job_name: 'mattermost'
+    static_configs:
+      - targets: ['localhost:8067']
```

Comment:
I am not sure this works. `localhost` from inside the container does not point to the host, hence the need for that special hostname.
I was able to make this work on my Linux setup by using `172.17.0.1` which is the default gateway. More info [here](https://dev.to/natterstefan/docker-tip-how-to-get-host-s-ip-address-inside-a-docker-container-5anh#:~:text=internal%20.&text=On%20Docker%20for%20Linux%2C%20the,you%20are%20using%20default%20networking.).